Assets/Scripts/Game/UserInterfaceWindows/DaggerfallQuestPopupWindow.cs

```diff
@@ -92,7 +92,7 @@ protected virtual void OfferQuest_OnButtonClick(DaggerfallMessageBox sender, Dag
 
                 // Show refuse message
                 sender.CloseWindow();
-                ShowQuestPopupMessage(offeredQuest, (int)QuestMachine.QuestMessages.RefuseQuest, false);
+                ShowQuestPopupMessage(offeredQuest, (int)QuestMachine.QuestMessages.RefuseQuest);
```

Comment:
Can you explain why this false has been removed please?

Reply:
The window was staying open after the user clicks “no” to an offered quest. I removed this and the CloseWindow on Setup so that the quest giver window closes itself properly regardless of whether the quest offer is shown after clicking “talk” or upon clicking the NPC.

Reply:
You've not used the latest version of this class for your work, so I would appreciate it if you could update to the latest version and test before you remove this false param.

Reply:
“You've not used the latest version of this class...”

Silly me. Thank you for catching that.

Reply:
“You've not used the latest version of this class...”

Silly me. Thank you for catching that.

Reply:
Good catch Hazelnut. I missed that one on the read.